assets/default.manifest

```diff
@@ -133,6 +133,41 @@ assets = (
     scale = 0.03125
     offset = [0.0,0.0,0.0]
 },
+{
+    name = "wire_1i"
+    type = "mesh"
+    file = "mesh/wire_1i.obj"
+    scale = 0.03125
+    offset = [0.0,0.0,0.0]
+},
+{
+    name = "wire_2i"
+    type = "mesh"
+    file = "mesh/wire_2i.obj"
+    scale = 0.03125
+    offset = [0.0,0.0,0.0]
+},
+{
+    name = "wire_4i"
+    type = "mesh"
+    file = "mesh/wire_4i.obj"
+    scale = 0.03125
+    offset = [0.0,0.0,0.0]
+},
+{
+    name = "wire_8i"
+    type = "mesh"
+    file = "mesh/wire_8i.obj"
```

Comment:
Why are these new meshes? Will offset not work?

Reply:
I don't want to use offset for this. It's for coping with needing different origin conventions, not for nudging things around by 1-2 voxels. These meshes may well also diverge soon -- I haven't done the 'L' filler pieces etc yet.

Reply:
Potential confusion with what I'd said in the toplevel comment: for entities i want the /entity/ to be placed differently -- not an asset manager level variant.